.pre-commit-config.yaml

```diff
@@ -47,7 +47,6 @@ repos:
             --disallow-any-generics,
             --pretty,
             --show-error-context,
-            --warn-unused-ignores,
```

Comment:
Why this change here, if it will be wholesale removed?  Seems like this will cause merge conflicts.

Reply:
This line is in the project toml too

https://github.com/pyvista/pyvista/blob/b516e91275eda3260c20a2e1cf0438a7997a6fce/pyproject.toml#L143-L153

Reply:
Yes this should be removed. It was an attempt to get CI to pass, since this is the only error raised by the pre-commit config. But, it seems pre-commit.ci is not catching this removal and is still warning about unused type ignores for some reason.